sell.py

```diff
@@ -1,102 +1,11 @@
-# Sell all ESPP shares from Computershare
-# Company name, login info, and an electronic payment already setup
+# Sell all ESPP shares from EquatePlus
+# login info, and an electronic payment already setup
 # Currently, an EFT avoids all fees and charges
 # Loading the previously mentioned info from a .env file in the same directory as this file
```

Comment:
For now, can we just leave this file alone for now? I started on fixing it but the new UI didn't let me get very far after I manually sold this first round.

Reply:
shall I revert any changes to it?

Reply:
Eh, nvm. I can just merge this as is and reapply on my end while I get through it in 2 weeks from now.